services/external/stripe/actions/delete-subscription.ts

```diff
@@ -0,0 +1,44 @@
+import { db } from "@/drizzle/db";
+import { subscriptions, teamMemberships } from "@/drizzle/schema";
+import { and, eq, ne } from "drizzle-orm";
+import { stripe } from "../config";
+
+export async function deleteSubscription(subscriptionId: string) {
```

Comment:
This method's name is a little bit misleading.
I think `handleSubscriptionDeletion` or similar is better here, because this method won't remove subscription.

Reply:
updated 4a7200794a53 ♻️